src/context.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from "react";
```

Comment:
please remove double quotes, semicolons and user same code indentation/spacing as in other files